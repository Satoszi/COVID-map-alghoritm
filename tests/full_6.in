40 40 4 117
full_6
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++
