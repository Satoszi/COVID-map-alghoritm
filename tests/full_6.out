39 39
34 39
29 39
23 39
17 39
12 39
6 39
0 39
26 37
9 37
3 37
31 36
14 36
35 35
19 35
39 34
11 34
6 34
1 34
27 33
22 33
16 33
32 32
9 31
4 31
35 30
29 30
24 30
19 30
14 30
0 30
39 29
11 28
6 28
32 27
27 27
22 27
17 27
2 27
37 26
14 25
9 25
34 24
29 24
24 24
19 24
5 24
0 24
39 23
16 22
11 22
32 21
27 21
22 21
7 21
2 21
37 20
19 19
14 19
34 18
29 18
24 18
10 18
5 18
0 18
39 17
21 16
16 16
32 15
27 15
12 15
7 15
2 15
37 14
24 13
19 13
34 12
29 12
15 12
10 12
5 12
0 12
39 11
26 10
21 10
32 9
17 9
12 9
7 9
2 9
37 8
29 7
24 7
34 6
20 6
15 6
10 6
5 6
0 6
39 5
31 4
26 4
22 3
17 3
12 3
7 3
2 3
36 2
29 1
39 0
33 0
25 0
20 0
15 0
10 0
5 0
0 0
